yvalenta/lib/client/client.rb

```diff
@@ -0,0 +1,34 @@
+require 'socket'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.